crates/gpui/src/platform/mac/text_system.rs

```diff
@@ -93,26 +93,18 @@ impl PlatformTextSystem for MacTextSystem {
     }
 
     fn all_font_names(&self) -> Vec<String> {
+        let mut names = Vec::new();
         let collection = core_text::font_collection::create_for_all_families();
         let Some(descriptors) = collection.get_descriptors() else {
-            return Vec::new();
+            return names;
         };
-        let mut names = BTreeSet::new();
```

Comment:
By the way, I removed the sorting and deduplication code, since we will handle these at the `TextSystem` level. Is this approach okay?